wagtailio/areweheadlessyet/management/commands/fetch_headless_issues.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from django.core.management.base import BaseCommand
+
+from wagtailio.areweheadlessyet.models import WagtailHeadlessIssue
+
+
+class Command(BaseCommand):
+    help = "Fetch issues from Wagtail's repository tagged #headless."
+
+    def handle(self, *args, **options):
+        response = requests.get(
+            "https://api.github.com/repos/wagtail/wagtail/issues",
+            params={"labels": "headless"},
+        )
+
+        if not response.ok:
+            message = response.json()["message"]
+            self.stdout.write(
+                self.style.ERROR(
+                    f"Fetching from Github's API failed for the following reason: {message}"
+                )
+            )
+            return
+
+        issues = response.json()
+        issues_in_db = set(
+            WagtailHeadlessIssue.objects.values_list("number", flat=True)
+        )
+        to_delete, to_create = ([], [])
+        for issue in issues:
+            if (issue_number := issue["number"]) in issues_in_db:
+                # Delete issue when it's closed.
+                if issue["state"] == "closed":
+                    to_delete.append(issue_number)
+            else:
+                to_create.append(
```

Comment:
Suggestion: I notice this doesn't handle title changes for issues. If that's something we want, it might be better to use `create_or_update` here. That will slow things down as it'll do `n` queries, but with the numbers we're dealing with, it shouldn't make a huge difference.

Reply:
Alright.